astropy/wcs/setup_package.py

```diff
@@ -324,6 +324,9 @@ def get_extensions():
     # do the copying here then include the data in [tools.setuptools.package_data]
     # in the pyproject.toml file
 
+    def requires_update(source: str, dest: str) -> bool:
+        return Path(source).stat().st_mtime > Path(source).stat().st_mtime
```

Comment:
@neutrinoceros - one should be `dest` I guess ;)

Reply:
of course, thanks !